.github/workflows/build.yml

```diff
@@ -4,26 +4,28 @@ on: [push]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Setup .NET Core
-      uses: actions/setup-dotnet@v1
-      with:
-        dotnet-version: 3.1.101
-    - name: Print openssl version
-      run: |
-        openssl version
-    - name: Decrypt signing key to use for strong-name
-      env:
-        SIGNINGKEY_ENC_PASSWORD: ${{ secrets.signingkeyEncPassword }}
-      run: |
-        openssl aes-256-cbc -salt -pbkdf2 -k "$SIGNINGKEY_ENC_PASSWORD" -in signingkey.snk.enc -out signingkey.snk -d    
-    - name: Install dependencies
-      run: dotnet restore
-    - name: Build
-      run: dotnet build --configuration Release --no-restore
-    - name: Test
-      run: dotnet test --no-restore --verbosity normal
+      - uses: digipost/set-timezone@v1.0
+        with:
+          timezoneLinux: "Europe/Oslo"
+      - uses: actions/checkout@v2
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 3.1.101
+      - name: Print openssl version
+        run: |
+          openssl version
+      - name: Decrypt signing key to use for strong-name
+        env:
+          SIGNINGKEY_ENC_PASSWORD: ${{ secrets.signingkeyEncPassword }}
+        run: |
+          openssl aes-256-cbc -salt -pbkdf2 -k "$SIGNINGKEY_ENC_PASSWORD" -in signingkey.snk.enc -out signingkey.snk -d
+      - name: Install dependencies
+        run: dotnet restore
+      - name: Build
+        run: dotnet build --configuration Release --no-restore
+      - name: Test
+        run: dotnet test --no-restore --verbosity d
```

Comment:
Sette denne tilbake til `normal`?
Fant du ut hva feilen var? Bra jobba 🙌🏻

Reply:
Jepp, var tydeligvis forskjell i hva som ble lest inn i `Subject` for sertifikatet 🤷  Lokalt funket det å hente ut orgnummer fra `OID.2.5.4.97`, mens det på Github Actions ble hentet ut som `organizationIdentifier`.

Reply:
Kan tyde på at streng-representasjonen som brukes til å hente ut info ikke er stabil; altså strengen kan endre seg i ulike .NET-versjoner og/eller implementasjon. Litt tilsvarende `X509Certificate.getSubjectDN()`-metoden i Java, som ikke anbefales å brukes nettopp p.g.a. dette, og [derfor endret](https://github.com/digipost/certificate-validator/commit/d2a304066086e9a1a08a6a1b63b1734e82d04a42) til å i stedet bruke `X509Certificate.getSubjectX500Principal` som gir streng-representasjon "by specification".

Reply:
Mhm, forsøkte å se om det fantes noe tilsvarende her, man klarte ikke finne det.